nym-wallet/src/components/Bonding/NodeStats.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { Stack, Typography, Box, useTheme, Grid, LinearProgress, LinearProgressProps } from '@mui/material';
+import { TBondedMixnode } from 'src/context';
+import { Cell, Pie, PieChart, Legend, ResponsiveContainer } from 'recharts';
+import { SelectionChance } from '@nymproject/types';
+import { NymCard } from '../NymCard';
+import { InfoTooltip } from '../InfoToolTip';
+
+const LinearProgressWithLabel = (props: LinearProgressProps & { value: number }) => {
+  const { value } = props;
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center' }}>
+      <Box sx={{ minWidth: 30 }}>
+        <Typography>{`${Math.round(value)}%`}</Typography>
+      </Box>
+      <Box sx={{ width: 40 }}>
+        <LinearProgress sx={{ borderRadius: 4 }} color="success" variant="determinate" {...props} />
+      </Box>
+    </Box>
+  );
+};
+
+const StatRow = ({
+  label,
+  tooltipText,
+  value,
+  progressValue,
+}: {
+  label: string;
+  tooltipText: string;
+  value: string | number;
+  progressValue?: number;
+}) => (
+  <Stack direction="row" gap={1} justifyContent="space-between" alignItems="center" width="100%">
+    <Stack direction="row" alignItems="center" gap={1} sx={{ color: (t) => t.palette.nym.text.muted }}>
+      <InfoTooltip title={tooltipText} />
+      <Typography>{label}</Typography>
+    </Stack>
+    {typeof progressValue === 'number' ? (
+      <LinearProgressWithLabel value={progressValue} />
+    ) : (
+      <Typography>{value}</Typography>
+    )}
+  </Stack>
+);
+
+const StatDivider = () => <Box borderTop="1px solid" borderColor="rgba(141, 147, 153, 0.2)" my={1} />;
+
+export const NodeStats = ({ mixnode }: { mixnode: TBondedMixnode }) => {
+  const {
+    stakeSaturation,
+    profitMargin,
+    estimatedRewards,
+    activeSetProbability,
+    standbySetProbability,
+    operatorCost,
+    routingScore,
+  } = mixnode;
+  const theme = useTheme();
+  const data = [
+    { key: 'routingScore', value: routingScore },
+    { key: 'rest', value: 100 - routingScore },
+  ];
+  const colors = [theme.palette.success.main, theme.palette.nym.nymWallet.chart.grey];
+
+  const getSetProbabilityLabel = (chance?: SelectionChance) => {
+    if (!chance) return 'Unknown';
+    switch (chance) {
+      case 'High':
+        return 'High';
+      case 'Good':
+        return 'Good';
+      case 'Low':
+        return 'Low';
+      default:
+        return 'Unknown';
+    }
+  };
+
+  const renderLegend = () => (
+    <Stack
+      alignItems="center"
+      maxWidth={200}
+      width={200}
+      sx={{
+        borderBottom: `1px solid ${theme.palette.nym.nymWallet.chart.grey}`,
+      }}
+    >
+      <Typography sx={{ color: (t) => t.palette.nym.text.muted }}>Routing score</Typography>
```

Comment:
When using sx prop in some cases (color, bgcolor, and some others) you don't need to access the theme this way. You can access the palette via a string `color: "nym.text.muted"`